client/signup/steps/woocommerce-install/step-business-info/revenue-options.js

```diff
@@ -0,0 +1,124 @@
+import formatCurrency from '@automattic/format-currency';
+import { __, _x, sprintf } from '@wordpress/i18n';
+
+// These are rough exchange rates from USD.  Precision is not paramount.
+// The keys here should match the keys in `getCurrencyData`.
+const exchangeRates = {
+	US: 1,
+	EU: 0.9,
+	IN: 71.24,
+	GB: 0.76,
+	BR: 4.19,
+	VN: 23172.5,
+	ID: 14031.0,
+	BD: 84.87,
+	PK: 154.8,
+	RU: 63.74,
+	TR: 5.75,
+	MX: 19.37,
+	CA: 1.32,
+};
+
+export function getCountryCode( countryState ) {
+	if ( ! countryState ) {
+		return null;
+	}
+
+	return countryState.split( ':' )[ 0 ];
+}
+
+export function getCurrencyRegion( countryState ) {
+	let region = getCountryCode( countryState );
+	const euCountries = [];
+	if ( euCountries.includes( region ) ) {
+		region = 'EU';
+	}
+
+	return region;
+}
+
+const getNumberRangeString = ( currency, min, max = 0 ) => {
+	if ( ! max ) {
+		return sprintf(
+			/* translators: store product count or revenue */
+			_x( '%s+', 'store product count or revenue', 'woocommerce-admin' ),
+			formatCurrency( min, currency )
+		);
+	}
+
+	return sprintf(
+		/* translators: store product count or revenue range */
+		_x( '%1$s - %2$s', 'store product count or revenue range', 'woocommerce-admin' ),
+		formatCurrency( min, currency ),
+		formatCurrency( max, currency )
+	);
+};
+
+const convertCurrency = ( value, country ) => {
+	const region = getCurrencyRegion( country );
+
+	if ( region === 'US' ) {
+		return value;
+	}
+
+	const exchangeRate = exchangeRates[ region ] || exchangeRates.US;
+	const digits = exchangeRate.toString().split( '.' )[ 0 ].length;
+	const multiplier = Math.pow( 10, 2 + digits );
+
+	return Math.round( ( value * exchangeRate ) / multiplier ) * multiplier;
```

Comment:
Can't figure out what are we trying to do with this formula but I am getting weird conversions, for example `up-to-2500` with Panama as a Country defaults to the US region and should return `up to 2500` but is returning `up to 3000`.

Reply:
it seems in the conversion it does `(2500 * 1) / 1000 = 2.5` and rounding that up would be `3` then `3 * 1000 = 3000`.

Reply:
Math isn't my strong suit but this looks like its doing a smoothing of the numbers so they don't look so random. It's rounding to a number of digits based on the exchange rate.

E.g. MX address,

**Without rounding** 
![Screenshot_2022-01-21_15-38-28](https://user-images.githubusercontent.com/811776/150467530-ad2e8965-592f-455b-9904-e0fa04c86bce.png)

**With rounding**
![Screenshot_2022-01-21_15-35-26](https://user-images.githubusercontent.com/811776/150467538-e12a6709-0f54-4bb1-a183-c034105918fa.png)

The defaulting to US region but still rounding is a bug, it should be returning `value` when that happens instead of applying the smoothing to the USD value.

The bigger issue with this is that we render values in `$` dollars instead of the currency for the value we've converted to (it shouldn't be using the locale currency symbol which we might be using?). We should probably also render USD instead of `$` when we fallback against it too.

Reply:
^ should all be fixed now.